### PR TITLE
(update:1502) - Updated the apps service with latest changes

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 1502

# Explain the change

Updated the apps service to latest graphql.

## Does this PR introduce a breaking change
Yes

## Screenshots

<details>
<summary>View Screenshots</summary>

image
</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?

### CODE REVIEW

**Comment by riginoommen** [comment]
@deshmukhmayur can you please review this

File: packages/apps-service/src/modules/apps/resolver.ts [hunk 1]
```diff
@@ -1,192 +1,198 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (deshmukhmayur) [inline packages/apps-service/src/modules/apps/resolver.ts:5]: What's the point of using eslint if the rules are disabled?

File: packages/apps-service/src/modules/common/index.ts [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// import { gql } from 'apollo-server';
```
Comment (deshmukhmayur) [inline packages/apps-service/src/modules/common/index.ts:1]: Remove the commented line.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

File: packages/apps-service/src/modules/apps/resolver.ts (hunk 1 above)
Comment (riginoommen) [reply packages/apps-service/src/modules/apps/resolver.ts]: updated

File: packages/apps-service/src/modules/common/index.ts (hunk 2 above)
Comment (riginoommen) [reply packages/apps-service/src/modules/common/index.ts]: it is removed and changed

**Review by ghanlohar — CHANGES_REQUESTED** [review]

File: packages/apps-service/src/index.ts [hunk 3]
```diff
@@ -1,65 +1,50 @@
...
+    onHealthCheck: (req) => Promise.resolve(req),
+    schema: makeExecutableSchema({
+      typeDefs: [CommonSchema, AppsSchema, MicroservicesSchema],
+      resolvers: [AppsResolver, MicroservicesResolver],
+    }),
     plugins: [
       {
-        requestDidStart: ( requestContext ) => {
-          if ( requestContext.request.http?.headers.has( 'x-apollo-tracing' ) ) {
+        requestDidStart: (requestContext): any => {
```
Comment (ghanlohar) [inline packages/apps-service/src/index.ts:25]:
```suggestion
        requestDidStart: (requestContext: any) => {
```

**Review by ghanlohar — APPROVED** [review]

File: packages/apps-service/package.json [hunk 4]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "apps-service",
-  "version": "1.0.0",
+  "version": "1.0.2",
```
[the remark this answers is not in the document]
Comment (riginoommen) [reply packages/apps-service/package.json]: Updated

File: packages/apps-service/src/@types/apps/index.d.ts [hunk 5]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
[the remark this answers is not in the document]
Comment (riginoommen) [reply packages/apps-service/src/@types/apps/index.d.ts:1]: I don't want to do a lot of changes now just configured the linter and fixed the basic lint issues

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**